src/test/java/com/zepben/evolve/services/network/translator/NetworkProtoToCimTestValidator.kt

```diff
@@ -106,14 +106,9 @@ class NetworkProtoToCimTestValidator(val network: NetworkService) {
         network.add(Breaker("breaker1"))
         network.add(Breaker("breaker2"))
 
-        pb.addEquipmentMRIDs("breaker1")
```

Comment:
If we are no longer adding the equipent (expected) do we still need to create the breakers above as I believe they are no longer used?